pkg/backporter/backporter.go

```diff
@@ -0,0 +1,296 @@
+package backporter
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+	"text/template"
+
+	"k8s.io/test-infra/prow/bugzilla"
+)
+
+const (
+	//BugIDQuery stores the query for bug ID
+	BugIDQuery = "ID"
+)
+
+const htmlPageStart = `
```

Comment:
This still needs to be moved to a `var` that gets populated via `template.Must`

Reply:
This is not a template, it is just a string. Does it still need to be moved in a template.Must? If so, why?

Reply:
ah okay, then its fine :)